server/controllers/stock/core.js

```diff
@@ -402,14 +408,38 @@ function stockManagementProcess(inventories) {
     Q = inventory.quantity; // the quantity
     CM = inventory.avg_consumption; // consommation mensuelle
```

Comment:
The `avg_consumption` is the CMM or AMC which is calculated during stock consumption (stock exit to patient or to service, or to a depot if the depot is a warehouse), implementation is in `/server/controllers/stock/index.js`